scripts/make_compmusic_raga_index.py

```diff
@@ -0,0 +1,175 @@
+import argparse
+import hashlib
+import json
+import os
+import glob
+from mirdata.validate import md5
+
+
+RAGA_DATASET_INDEX_PATH = "../mirdata/datasets/indexes/compmusic_raga_index.json"
+
+
+def make_otmm_makam_index(dataset_data_path):
+
+    raga_index = {"version": "1.0", "tracks": {}, "metadata": []}
+    dataset_folder = "RagaDataset"
+
+    traditions = ["Carnatic", "Hindustani"]
+    for trad in traditions:
+        tradition_name = trad
+        for raga in glob.glob(os.path.join(dataset_data_path, trad, "audio", "*/")):
+            raga_id = raga.split("/")[-2]
```

Comment:
os.path.sep instead of "/"? otherwise this may fail on windows

Reply:
that is true! I'll change that.

Reply:
We will actually address that given the refactor we need to do to fix path issues with windows all over mirdata.